sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtil.java

```diff
@@ -22,9 +22,9 @@
   static final String DATA_TYPE_METRICS = "metrics";
 
   static String getOtlpProtocol(String dataType, ConfigProperties config) {
-    String protocol = config.getString("otel.experimental.exporter.otlp." + dataType + ".protocol");
```

Comment:
should we continue to support the `experimental` versions for one release, for those who might be using them?

Reply:
Sure that seems reasonable.

Reply:
I would have them documented, but clearly marked as deprecated in the docs.